templates/pan-cn-mgmt.yaml

```diff
@@ -117,11 +117,11 @@ spec:
           resources:
             requests:
               # configurable based on desired logging, capacities
-              cpu: "{{ .Values.mp.cpu }}"
-              memory: "{{ .Values.mp.mem }}"
+              cpu: "2"
+              memory: "2.0Gi"
```

Comment:
why hard code the CPU/mem here?

Reply:
Each YAML file had a request and limit but the definitions were pretty narrow in the PDF guide.  So I left the request and limit at the min/max values and allowed a variable for overriding the vCPU request value.  Won't k8s scale up from request to limit if needed?

Reply:
In other words, the documentation had no defined limit for the number of CPUs.  So I made a cpuLimit variable and hard coded the required minimum.  The doc had min/max values for memory so I plugged those into the templates directly.